omnibus/Gemfile.lock

```diff
@@ -465,8 +465,8 @@ GEM
 
 PLATFORMS
   ruby
-  x64-mingw
   x64-mingw32
+  x64-unknown
```

Comment:
@chef/chef-workstation-approvers : This does not seem to be correct. Or is it ?  any thoughts ?